src/cli.ts

```diff
@@ -148,6 +149,14 @@ void yargs
         return new SendTextCommand().run(args);
     })
 
+    .command('interactive', 'Provides a way to send a series of ECP key events similar to how Roku Remote Tool works but from the command line', (builder) => {
```

Comment:
Chris and I played around with some alternative names. How about:

`remote-control` 

So usage would be:

```
npx roku-deploy remote-control
```

We could add an alias like `RC` or `R` or something if that feels verbose?

Reply:
updated. Added `rc` shorter version as well